composer.json

```diff
@@ -11,14 +11,14 @@
     }
   ],
   "require": {
-    "php": "^7.4|^8.0|^8.1",
+    "php": "^7.4|^8.0|^8.1|^8.2",
```

Comment:
Shouldn't be: `"php": "^7.4|^8.0"`? (that covers all 8.x.x versions)

Reply:
Yes, the caret should cover until the next major version. I wanted to be conservative but let's remove it :+1: